server/src/main/java/com/orientechnologies/orient/server/OServer.java

```diff
@@ -533,6 +538,14 @@ protected boolean deinit() {
           OLogManager.instance().info(this, "Shutting down databases:");
           Orient.instance().shutdown();
         } catch (Throwable e) {
+          // We are eating all exceptions here. As a result, all state verification logic is unable to fail the tests. At least
+          // AssertionError-s should be passed through, or even better, all exceptions if we are running in a testing env
+          // (check for -ea?).
+          //
+          // Another thing to think about, in-process server tests will fail with pass-through exceptions, but tests running a

```

Comment:
If I remember correctly now all tests use in-process servers, so we can skip this problem for now at least.